packages/sprotty/src/features/move/move.ts

```diff
@@ -106,16 +106,32 @@ export class MoveCommand extends MergeableCommand implements IStoppableCommand {
                 if (resolvedMove) {
                     this.resolvedMoves.set(resolvedMove.element.id, resolvedMove);
                     if (this.edgeRouterRegistry) {
-                        index.getAttachedElements(element).forEach(edge => {
-                            if (edge instanceof SRoutableElementImpl) {
-                                const existingDelta = attachedEdgeShifts.get(edge);
-                                const newDelta = Point.subtract(resolvedMove.toPosition, resolvedMove.fromPosition);
-                                const delta = (existingDelta)
-                                    ? Point.linear(existingDelta, newDelta, 0.5)
-                                    : newDelta;
-                                attachedEdgeShifts.set(edge, delta);
+                        const handleEdges = (el: SModelElementImpl) => {
+                            index.getAttachedElements(el).forEach(edge => {
+                                if (edge instanceof SRoutableElementImpl) {
+                                    const existingDelta = attachedEdgeShifts.get(edge);
+                                    const newDelta = Point.subtract(resolvedMove.toPosition, resolvedMove.fromPosition);
+                                    const delta = (existingDelta)
+                                        ? Point.linear(existingDelta, newDelta, 0.5)
+                                        : newDelta;
+                                    attachedEdgeShifts.set(edge, delta);
+                                }
+                            });
+                        };
+                        const handleEdgesForChildren = (el: SModelElementImpl) => {
```

Comment:
Do we really need to move all edges for children? If an edge is contained in the element, its coordinates are usually relative to that element, so we don't need to change them when the element is moved.

Reply:
Yes. They need to be put into `attachedEdgeShifts` because otherwise they aren't handled [here](https://github.com/eclipse-sprotty/sprotty/pull/442/files/6039b06761187ed10c2ca1a6f7dde79f2fccc49a#diff-6c61444caead7afcf404c2d10422199a977d531c2e3cb23625a0cd1af1d2a4d4R187) and the routing points stay where they are.

Reply:
But what happens if package A contains nodes B and C, and those two nodes are connected by an edge which is also a child of A? Could you try that case? What we need to avoid is moving routing points for elements that are relative to a moved container.